app/assets/stylesheets/mixins/_forms.scss

```diff
@@ -124,5 +124,17 @@
       }
     }
   }
-}
 
```

Comment:
I wasn't sure if this would be the file to add this code. I'm happy to move it on your suggestion.